.stickler.yml

```diff
@@ -1,11 +1,20 @@
 ---
 linters:
-    flake8:
-    eslint:
-        config: './eslint.config'
-    csslint:
-
+  csslint: {  }
 files:
-    ignore:
-        - 'src/generated/*'
-        - 'webroot/bower_components/*'
+  ignore:
+    - 'bower_components/*'
+
+
+linters:
```

Comment:
We don't need `phpcs` linter as it is the linter for php. We just need to enable `style lint` [here](https://github.com/microverseinc/linters-config/tree/master/css)